This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marco Borrini <borrinimarco74@gmail.com>
Date: Tue, 4 Aug 2026 23:08:00 +0200
Subject: [PATCH 1/1] Fix LC_MESSAGES on MinGW

MinGW does not define LC_MESSAGES. Fallback to LC_ALL.

diff -ur canberra-orig/src/sound-theme-spec.c canberra-new/src/sound-theme-spec.c
--- canberra-orig/src/sound-theme-spec.c
+++ canberra-new/src/sound-theme-spec.c
@@ -34,6 +34,10 @@
 #include "llist.h"
 #include "cache.h"
 
+#ifndef LC_MESSAGES
+#define LC_MESSAGES LC_ALL
+#endif
+
 #define DEFAULT_THEME "freedesktop"
 #define FALLBACK_THEME "freedesktop"
 #define DEFAULT_OUTPUT_PROFILE "stereo"
